version/version.go

```diff
@@ -12,3 +17,11 @@ var (
 	// the program at linking time.
 	Revision = ""
 )
+
+func String() string {
```

Comment:
nit: There's other user agents throughout the code base (e.g. `cmd/version.go`), we should switch them all to this

Reply:
https://github.com/dagger/dagger/pull/2527